src/vuestic-theme/vuestic-components/va-input/VaInput.vue

```diff
@@ -116,6 +117,28 @@ export default {
     removable: {
       type: Boolean,
     },
+
+    // textarea-specific
+    autosize: {
+      type: Boolean,
+      default: false,
+    },
+    minRows: {
+      type: Number,
+      validator: (val) => val > 0 && (val | 0) === val,
```

Comment:
Validation in props is not too descriptive (it just says validation failed). For complex cases it's better to ``throw new Error(`With detailed description of ${what} is wrong.`)``.